router.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"embed"
+	"encoding/json"
+	"fmt"
+	"github.com/labstack/echo/v4"
+	middleware2 "github.com/labstack/echo/v4/middleware"
```

Comment:
Could this be something more meaningful than `middleware2` (eg `echoMw` as in `middleware/middleware.go`)?

Reply:
Good point, I have a habit of adding a number after something that doesn't work exactly how I want it to